documentation/modules/uninstalling-mtv-cli.adoc

```diff
@@ -6,31 +6,38 @@
 [id="uninstalling-mtv-cli_{context}"]
 = Uninstalling {project-short} from the command line interface
 
-You can uninstall {project-first} from the command line interface (CLI) by deleting the +{namespace}+ project and the `forklift.konveyor.io` custom resource definitions (CRDs).
+You can uninstall {project-first} from the command line interface (CLI) by deleting the `forklift` controller, the {project-short} Operator, and the plugin console custom resource (CR).
```

Comment:
Suggestion - In the abstract, do not replicate the steps that will be covered in the procedure.

Reply:
OK.